deployment-detector/src/utils.ts

```diff
@@ -1,33 +1,37 @@
-import { Finding, FindingSeverity, FindingType } from "forta-agent";
-export const NETHERMIND_DEPLOYER:string = "0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8";
-export const FORTA_PROXY_ADDRESS:string = "0x61447385B019187daa48e91c55c02AF1F1f3F863"
-export const FORTA_CREATE_AGENT:string = "function createAgent(uint256 agentId,address owner,string metadata,uint256[] chainIds)"
+import {  Finding, FindingSeverity, FindingType } from "forta-agent";
```

Comment:
```suggestion
import { Finding, FindingSeverity, FindingType } from "forta-agent";
```
Small fix, unnecessary space in import